test/Math.t.sol

```diff
@@ -74,62 +71,8 @@ contract DivupSuccessTests is MathTestBase {
 
 }
 
-contract RPowFailureTests is MathTestBase {
```

Comment:
Add a note that since SSR bounds are MAX_SSR, these failure tests aren't necessary